index.js

```diff
@@ -6,15 +7,258 @@ function ask(questionText) {
+  }
+  let nextStep = await ask('\n\nShould you land on the ship?\nType "land" or "fly away"? ')
+  if (nextStep !== 'land') {
+    console.log('\n\nWell, your calculations were wrong about how much remaining fuel your ship had..\n\nYou have succumbed to the vast, cold emptiness of space.')
+    process.exit()
+  } else {
+    console.log('\nYou touch down on the ship\'s landing pad successfully!')
+  }
+  console.log('\n\nYou notice a light flickering above the dimly lit hangar, and as you continue your initial obeservations, it becomes quite apparent that the ship has lost power!\n\n!!!The ship is sinking closer to a black hole!!!\n\nYou must act quickly and search this cruiser to find the ECM your ship needs\nbefore the black hole consumes you and the cruiser!!!\n\n\nYou can change rooms by typing "move"\nYou can take an item for your inventory by typing "take".\nYou can drop an item by typing "drop".\nTo check your inventory type "check inventory"\nTo check your status type "status".\nTo read an item type "read"\n\n***DISCLAIMER*** This rollercoaster doesn\'t have any seatbelts...\n Please type user inputs exactly as they are listed or\n you might prematurely end your adventure!')
+
+  let blueprintQ = await ask('\nYou find a blueprint of the ship.\nThe blueprint says the ship\'s spare parts are in the Engine Block.\ndo you take or leave. ')
+  if (blueprintQ === 'leave') {
+    console.log('\n***You place the blueprint back down***')
+  } else {
+    console.log('\nOh no, the poison left on the blueprint spreads throughout your body\nyou have now died\nINCONCEIVABLE! ')
+    process.exit()
+  }
+
+
+  console.log((roomsLookUp[player.location].desc))
+  //---------------- Main Story code: move, take, drop, status, inventory ----------
+  while (true) {
+    let room = roomsLookUp[player.location];
+    let question = await ask('move, drink, take, drop, check inventory, read, status\nWhat do you want to do? ')
+    if (question === 'move') {
+      console.log(shipRooms[player.location].canChangeTo)
+      let clarifyingQ = await ask('\nWhat room do you want to go to? ')
+      if (clarifyingQ === 'engineBlock' && engineBlock.locked === true) {
+        await passcodeEngineBlock()
+      }
+      if (clarifyingQ === 'hangarControlRoom') {
+        turnPowerOnInHangarControlRoom()   
+      }
+      room.moveBetweenRooms(clarifyingQ)
+      room = roomsLookUp[player.location]
+      console.log(room.desc)
+    }
+    else if (question === 'take') {
+      let takeQ = await ask('\nWhat would you like to take? ')
+      if (room.inv.includes(takeQ)) {
+        console.log('\nYou have taken this item for your inventory.')
+        player.inventory.push(takeQ)
```

Comment:
you can add a `room.inventory.pop(item)` to remove the item from the room here, so the user can no longer take multiple of the same item.

but for flavor, you can have them keep taking the energy drinks since it says there is a case of them.

Reply:
`pop` can't be used to remove a specific item, you need `splice` for that.